### ISSUE TITLE
Add Assignment feedback note

### ISSUE THREAD

**Issue by splice-7** [Aspine/aspine#359]
![image](https://user-images.githubusercontent.com/118221547/235155711-85dceb3b-68e2-48f7-b335-ae7760393748.png)
Scrape these notes that teachers (rarely) give and display them on Aspine's grades page. 

<sub>(my spanish teacher said to not use aspine cause it doesn't show notes and that's bad for aspine's reputation)</sub>

**Comment by tektaxi** [comment]
We actually do already show the assignment notes! If you click on the info (i) icon next to the assignment it pulls up all the assignment details, including the note (called "assignment feedback"). It defaults to none but if there is a note it will display there. You can tell your Spanish teacher that! It's possible that displaying it in a more accessible way would be a better option, and we can explore that possibility.

<img width="1406" alt="Screenshot 2023-04-28 at 12 58 19" src="https://user-images.githubusercontent.com/45189395/235208506-d470f294-beaf-4849-bd7c-65789edabebf.png">
^^ you can see the assignment feedback section here

**Comment by splice-7** [comment]
Oh. Nevermind this then. 

...or maybe if there is a note, make that assignment's info icon a different color (or something else that would make it stand out) to indicate that there is a note.

**Comment by jshaw-7** [comment]
yeah, I think a lot of people don't know that you can see more information on an assignment. maybe we could add text to the little 'i' icon (like 'assignment information' or something), to make it more obvious what it does? same with the text corrections button- not a lot of people know that it exists.
